dev-vcs/bzr/patches/bzr-2.7.0.patchset

```diff
@@ -0,0 +1,91 @@
+From 26c866e6de99f4129d62ecfca552958af0a8ef18 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Fran=C3=A7ois=20Revol?= <revol@free.fr>
+Date: Mon, 4 Jun 2018 09:14:57 +0200
+Subject: Use gcc2 zero-length array member
+
+
+diff --git a/bzrlib/diff-delta.c b/bzrlib/diff-delta.c
+index 0801c96..a0a8dc5 100644
+--- a/bzrlib/diff-delta.c
++++ b/bzrlib/diff-delta.c
+@@ -24,6 +24,12 @@
+ #include <string.h>
+ #include <assert.h>
+ 
++# if defined(__GNUC__) && (__GNUC__ < 3)
++#  define FLEX_ARRAY_LEN 0
```

Comment:
I'm replying to my previous message:
> BTW, you might wish to insert a few spaces to fix the indentation. The result would look like:

Oops, sorry, it looks like my suggestion to fix the indentation by adding spaces was not a good idea.
What I should have suggested was to remove the first space in `# if defined(__GNUC__) && (__GNUC__ < 3)`.

So, instead of:
```diff
 #include <assert.h>
 
+# if defined(__GNUC__) && (__GNUC__ < 3)
+#  define FLEX_ARRAY_LEN 0
+# else
+#  define FLEX_ARRAY_LEN
+# endif
```
I think the indentation would look nicer like this:
```diff
 #include <assert.h>
  
+#if defined(__GNUC__) && (__GNUC__ < 3)
+# define FLEX_ARRAY_LEN 0
+#else
+# define FLEX_ARRAY_LEN
+#endif
```
(Sorry about this, btw... I had not noticed that there was no space between the `#` and the `include <assert.h>` in line 13.)

Reply:
Thought it looked weird ;) (but didn't produce an error) ;)